src/ess/powder/correction.py

```diff
@@ -335,6 +338,22 @@ def _shallow_copy(da: sc.DataArray) -> sc.DataArray:
     return out
 
 
+def subtract_background(
+    data: FocussedDataDspacing[SampleRun],
+    background: FocussedDataDspacing[BackgroundRun],
+) -> BackgroundSubtractedData[SampleRun]:
+    return BackgroundSubtractedData[SampleRun](data.bins.concatenate(-background))
```

Comment:
Will we need the background subtraction to work also for histogrammed data?

Reply:
I don't know. For now, I have not seen histogrammed data at this point in the workflow.